fontforge/fvfonts.c

```diff
@@ -1310,14 +1310,11 @@ return( head );
 }
 
 static void InterpPoint(SplineSet *cur, SplinePoint *base, SplinePoint *other, real amount ) {
```

Comment:
A general comment on these: If you allocate a new point and don't call SplineMake, or change an existing point and don't call SplineRefigure, and then put the point "back" into normal use, those are bugs. In the first case the points won't really be part of a common contour because its SplineMake that links them up. In the latter case you'll just get bad values because almost all calculations use the expanded coefficients.

For this one I don't see an issue. The case where SplineMake isn't called is when adding the first point to the SplineSet. Sure, the values are adjusted while nonextcp and noprevcp will retain the default `false` values from `SplinePointCreate()`, but the metaphysics of a dangling control point are unclear anyway. The worst that could happen is loosing the dangling coordinates on a round-trip, but I don't believe that's a problem with SFDs. (When round-tripping through external tools the result will be highly variable, again because of the unclear metaphysics.